lib/elixir/lib/calendar/iso.ex

```diff
@@ -1,22 +1,52 @@
 defmodule Calendar.ISO do
   @moduledoc """
-  A calendar implementation that follows to ISO 8601.
+  The default calendar implementation, a Gregorian calendar following ISO 8601.
```

Comment:
a proleptic Gregorian calendar following .....

Reply:
I wasn't sure if `proleptic` was super helpful for the summary description, wdyt?

Reply:
Yes, you are right, I'm being too pedantic give it is mentioned in the next line (which I hadn't read at the time)